### PR TITLE
Add fold-none chunk option to opt-out code folding from individual chunks in html_document

### PR DESCRIPTION
**Pull request by atusy**
solves #2346 

- implement fold-none option
- update documents
- devtools::document()

### CODE REVIEW

File: R/html_document.R [hunk 1]
```diff
@@ -287,6 +284,24 @@
 #'  to via mouse clicks.} \item{\code{print} (defaults to \code{TRUE}) controls
 #'  whether the table of contents appears when user prints out the HTML page.}}
 #'
+#'@section Code folding:
```
Comment (yihui) [inline R/html_document.R:287]: I don't know why we omitted the spaces before all `@section`  tags in this file, but I'd like to add them back.

```suggestion
#' @section Code folding:
```

**Review by yihui — APPROVED** [review]
Looks good. Thanks!

**Comment by atusy** [comment]
@yihui Thanks. I added spaces to other @-marks, too.

**Review by cderv — APPROVED** [review]
Awesome thanks !

**Pull request merged**